src/pages/en/guides/markdown-content.md

```diff
@@ -273,6 +273,22 @@ With the `@astrojs/mdx` integration, you can use Astro or UI framework component
 
 Don't forget to include a `client:directive` if necessary!
 
+```astro title="src/pages/about.mdx"
```

Comment:
The syntax here might need updating. I wasn't sure what to put for the `{3,6}` stuff, etc

Reply:
I’d suggest this here:

````suggestion
```astro title="src/pages/about.mdx" {5-6} /<.+\/>/
````

- `{5-6}` will highlight lines 5 to 6 where the import statements are.
- `/<.+\/>/` is regular expression that will match both `<Button />` and `<ReactCounter />` and highlight the usage.

Reply:
Thanks for the review and explanation @delucis ❤️